config/v201/core_migrations/1_up-initial.sql

```diff
@@ -86,3 +86,9 @@ CREATE TABLE METER_VALUE_ITEMS (
     ENCODING_METHOD TEXT,
     PUBLIC_KEY TEXT
 );
+
+CREATE TABLE CHARGING_PROFILES (
```

Comment:
Like mentioned in the weekly call, please create a new up (and down) file for these changes.

Reply:
What should be in the `down` file? We don't want to delete profiles when we shut down, right?

Reply:
The down file is only used when an older version of libocpp is installed so that we can go back to the exact schema that version expects. If you add a table like you are doing here, the down file should just drop the table.

See https://github.com/EVerest/libocpp/blob/main/doc/database_migrations.md